projects/frontend/.env.sample

```diff
@@ -0,0 +1 @@
+VITE_API_URL="http://localhost:3000"
```

Comment:
Could you add this env var in the dockerfile/fly.toml to include it during the frontend build?